src/main/webapp/WEB-INF/user.jsp

```diff
@@ -41,11 +52,17 @@
     <% }  %>
 
     <% if(isUserLoggedIn){ %>
-    <form id="message-form" action="/messages?recipient=<%= user %>" method="POST" class>
+    <form id="message-form" action="/messages?recipient=<%= user %>" method="POST" enctype="multipart/form-data" class>
     Enter a new message:
     <br/>
     <textarea name="text" placeholder="Enter a message" id="message-input"></textarea>
     <br/>
+    Add an image to your message:
+    <input type="file" name="image">
+    <% const messageForm = document.getElementById('message-form'); %>
```

Comment:
I'm pretty sure this is not right. It looks like you are trying to use JavaScript inside JSP tags that expect Java. I'm also a little confused why you are overriding the form action here instead of just modifying it in the <form> above. 

Should this be in its own form instead?

Reply:
So I was trying just to make it work because I really don't understand a lot of JSP and because I haven't found a way to do it that's why I did it like that. The main problem is that there was a method called load message in which I was supposed to add code so this could work, I think in the change to JSP just disappeared so right now I'm trying to find another way to do it.